Engineering/BestPractices/DoneDone.md

```diff
@@ -4,9 +4,12 @@ Software Engineering is not just implementing functionality and writing code, bu
 
 Below is an example Done-Done checklist you should follow to ensure everything is done (may vary depending on project/client):
 * Is covered by unit and integration tests YES/NO
+* Is linted and meets the CSE coding (by language) best practices YES/NO
+* Agile activites like sprint reviews, retrospectives are completed YES/NO
 * Is continuously tested in each environment YES/NO
-* Is sufficient diagnostic/user telemetry and dashboarding YES/NO
-* Is sufficient public documentation YES/NO
+* Has sufficient diagnostic/user telemetry and dashboarding YES/NO
+* Has sufficient public documentation YES/NO
+* Technical Handover documentation soundboarded with "third-party" internal resource for readability and coherence YES/NO
```

Comment:
By saying `"third-party" internal resource`, is line 12 saying "get your handover documentation reviewed by at least X people in CSE"?  It might be clearer to just say that.

Reply:
Agreed on rephrasing but maybe not necessarily within CSE, we might have someone who is better suited for a particular scenario - example being MSR group for some ML related project. Thoughts?